crates/router/src/db/address.rs

```diff
@@ -91,38 +92,116 @@ impl AddressInterface for Store {
 
 #[async_trait::async_trait]
 impl AddressInterface for MockDb {
+    #[allow(clippy::unwrap_used)]
```

Comment:
Remove all the `#[allow(clippy::unwrap_used)]` annotations added for all methods in this implementation. We don't need it anymore, since we don't `unwrap()` here.

Also, please address the CI checks.